tests/framework/microvm.py

```diff
@@ -993,6 +993,11 @@ def open(self):
             # serial already opened
             return
 
+        attempt = 0
+        while not Path(self._vm.screen_log).exists() and attempt < 5:
+            time.sleep(0.2)
+            attempt += 1
```

Comment:
Is it only enough for solving the intermittent issues that we see to have the `Serial` opened, or do we need to wait for some output at it? Would it make sense to _also_ wait for the microVM to boot (by running something like `vm.ssh("true")`?

Reply:
The problem with that is the cmdline test uses `init=/bin/cat -- /etc/issue`, which I doubt will let us connect via ssh.

Reply:
then what about a `sleep` after booting the microVM (additional to the sleep for the Serial output)?

Reply:
It feels like it's already covered by the pattern matching code: https://github.com/firecracker-microvm/firecracker/blob/main/tests/framework/microvm.py#L982 . If the file exists, it should take care of that.